packages/client/tests/functional/accelerate-without-no-engine-hint/tests.ts

```diff
@@ -0,0 +1,39 @@
+import { NewPrismaClient } from '../_utils/types'
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let newPrismaClient: NewPrismaClient<typeof PrismaClient>
+
+testMatrix.setupTestSuite(
+  (suiteConfig, suiteMeta, clientMeta) => {
+    testIf(clientMeta.dataProxy === false)('using accelerate without --no-engine produces a warning at runtime', () => {
+      process.env[`DATABASE_URI_${suiteConfig.provider}`] = 'prisma://accelerate.net/?api_key=doesNotMatter'
+      const consoleWarnMock = jest.spyOn(console, 'warn').mockImplementation()
+
+      newPrismaClient()
+
+      expect(consoleWarnMock).toHaveBeenCalledTimes(1)
+      expect(consoleWarnMock.mock.calls[0]).toMatchInlineSnapshot(`
+        [
+          In production, we recommend using \`prisma generate --no-engine\` (See: \`prisma generate --help\`),
```

Comment:
Users will get this suggestion as a warning if they are not using `--no-engine`.